src/Core/Metadata/Delete.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Metadata;
+
+#[\Attribute(\Attribute::TARGET_CLASS | \Attribute::IS_REPEATABLE)]
+class Delete extends Operation
+{
+    public function __construct(...$values)
```

Comment:
This breaks the auto-completion. Maybe could we find a PHPDoc trick to hint the IDE to use the auto-completion of the parent class.

Reply:
Same for other operations